pkgs/tools/misc/disper/default.nix

```
@@ -1,9 +1,17 @@
-{stdenv, fetchurl, python, xorg, makeWrapper}:
+{ stdenv, fetchFromGitHub, python, xorg, makeWrapper }:
 
 stdenv.mkDerivation rec {
-  name = "disper-0.3.1";
+  pname = "disper";
+  version = "0.3.1.1";
 
-  buildInputs = [python makeWrapper];
+  src = fetchFromGitHub {
+    owner = "apeyser";
+    repo = pname;
+    rev = "${pname}-${version}";
+    sha256 = "1kl4py26n95q0690npy5mc95cv1cyfvh6kxn8rvk62gb8scwg9zn";
+  };
+
+  buildInputs = [ python makeWrapper ];
```

```suggestion
  nativeBuildInputs = [ makeWrapper ];
  buildInputs = [ python ];
```

Even though this is not part of your change, could you please move `makeWrapper` to `nativeBuildInputs`? (1)
Otherwise this looks good and also works fine :+1:


(1): https://nixos.org/nixpkgs/manual/#ssec-stdenv-dependencies

---

now would be the time to do it :heart:  sure.